### DISCUSSION TITLE
Import Polar H10 HR and RR .txt files

### DISCUSSION THREAD

**Discussion by skelly0823** [mytourbook/mytourbook#1138] [Q&A]
I'd like to use MTB's Merge Tour capability to compare HR data between Garmin .FIT files and Polar H10 Sensor data from .TXT files.   I've searched the forums but cannot find a good solution for importing .TXT data. The H10 .TXT files can be converted to .csv for custom import but I don't see where a HR data field can be inserted.   Is converting the H10 .TXT file to the Polar .HRM format the best solution?

Thanks in advance for your prompt reply, SK 

attached;

[Polar_H10_96EF7025_20220312_074947_HR.txt](https://github.com/mytourbook/mytourbook/files/11888227/Polar_H10_96EF7025_20220312_074947_HR.txt)
[Polar_H10_96EF7025_20220312_074947_RR.txt](https://github.com/mytourbook/mytourbook/files/11888229/Polar_H10_96EF7025_20220312_074947_RR.txt)

**Comment by FJBDev** [comment]
SK,

As you want to compare HR data between FIT and H10 txt file, you could import your fit files in MT and export them in CSV format to then compare with the H10 file?

**Reply by skelly0823** [reply]
Yes, I could compare the data external to MTB, however there is a lot of H10 data that I'd like to import to MTB anyway. It would be great to be able to create custom csv files with time slices and HR or other sensor data. Alternatively I can convert all the H10 data to one of the existing file formats. Not sure which file format would be easiest to convert to, maybe one of the .xml formats.

**Comment by FJBDev** [comment]
>  Alternatively I can convert all the H10 data to one of the existing file formats. Not sure which file format would be easiest to convert to, maybe one of the .xml formats.

While I am still not sure to understand why your Polar H10 generates a txt file instead of merging the HR data to your device (with GPS, distance...), here is what you could do:
Convert the H10 txt file to a TCX file and import it into MT

Here is an example of what a line in the txt file would be converted in a TCX file:

`2022-03-12T07:49:57.638;984`


=> 


```
<Trackpoint>
<Time>2022-03-12T07:49:57.638</Time>
<HeartRateBpm>
	<Value>XX</Value>
</HeartRateBpm>
</Trackpoint>

```
